From 539a34e463a8117a3a0dcb8954fb4b406991a53b Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Mon, 17 Apr 2023 12:14:21 +0100
Subject: [PATCH 1/3] gtk: Add release history to metainfo file

This is mandatory on Flathub (and possibly other software repositories).
On each new release, a <release> element should be prepended to this
list.

Ideally each <release> would include (translatable) release notes, but
this is the bare minimum.

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-releases
https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Releases.html
---
 gtk/transmission-gtk.metainfo.xml.in | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/gtk/transmission-gtk.metainfo.xml.in b/gtk/transmission-gtk.metainfo.xml.in
index e39180776..fb7d59dfc 100644
--- a/gtk/transmission-gtk.metainfo.xml.in
+++ b/gtk/transmission-gtk.metainfo.xml.in
@@ -38,4 +38,11 @@ Copyright 2017 Endless Mobile, Inc.
   <content_rating type="oars-1.1">
     <content_attribute id="social-info">mild</content_attribute>
   </content_rating>
+  <releases>
+    <release date="2023-04-14" version="4.0.3" />
+    <release date="2023-03-16" version="4.0.2" />
+    <release date="2023-02-23" version="4.0.1" />
+    <release date="2023-02-08" version="4.0.0" />
+    <release date="2020-05-03" version="3.00" />
+  </releases>
 </component>
-- 
2.30.2

